### PR TITLE
Anchor links not working for dynamically loaded anchors #1241

### LINKED ISSUE #1241: Anchor links not working for dynamically loaded anchors
See https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-id3-id5-readers#protein-detection-quantitation-and-analysis

This should deep link to the applications tab to the correct applications. It works from the applications tabs toc but not on the very first load of the page. The application list is loaded dynamically and therefor the hash does not exist on the initial html. We need some JS to process this after the block has loaded.

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #1241

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/microplate-readers/multi-mode-readers/spectramax-id3-id5-readers#protein-detection-quantitation-and-analysis
- After: https://1241-fix--moleculardevices--hlxsites.hlx.page/products/microplate-readers/multi-mode-readers/spectramax-id3-id5-readers#protein-detection-quantitation-and-analysis

- Before: https://main--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids#cardiac-organoids
- After: https://1241-fix--moleculardevices--hlxsites.hlx.page/applications/3d-cell-models/organoids#cardiac-organoids

- Before: https://main--moleculardevices--hlxsites.hlx.page/about-us#we-design-complete-solutions-for-any-application-to-meet-customer-needs-today--and-tomorrow
- After: https://1241-fix--moleculardevices--hlxsites.hlx.page/about-us#we-design-complete-solutions-for-any-application-to-meet-customer-needs-today--and-tomorrow

### CODE REVIEW

**Comment by lydiapuric** [comment]
TBT increased because we need to wait til element exists. Checking if I can improve this.

**Comment by lydiapuric** [comment]
Increasing timeout to 3500 when detecting anchor improves TBT and brings back LHS to 75

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -181,6 +181,34 @@ export function decorateIcons(element = document) {
   });
 }
 
+/**
+ * Detect anchor
+ */
+export function detectAnchor(block) {
```
Comment (mhaack) [inline scripts/lib-franklin.js:187]: This should be in scripts.js not lib-franklin.js

Comment (lydiapuric) [reply scripts/lib-franklin.js:187]: Adjusting

Comment (lydiapuric) [reply scripts/lib-franklin.js]: Done

**Pull request merged**